packages/snake/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/TipzTeam/snake-cpp-termux
+TERMUX_PKG_DESCRIPTION="Eat as much as you want while avoiding walls."
+TERMUX_PKG_LICENSE=MIT
+TERMUX_PKG_VERSION=1.0
+TERMUX_PKG_SRCURL=https://github.com/TipzTeam/snake-cpp-termux/archive/${TERMUX_PKG_VERSION}.tar.gz
```

Comment:
Forks shouldn't be used unless there is a good reason, e.g. original owner transferred project's ownership.

If source requires some changes to get working in Termux, this should be done through patches placed in directory where build.sh is located.

If project doesn't specify a version (https://github.com/gogakoreli/snake/ doesn't specify it), then you should use a date of latest commit in format YYYY.MM.DD and in source URL a git commit hash should be used.